tests/php/MetaboxTest.php

```
@@ -0,0 +1,215 @@
+
+	/**
+	 * Capture the output of a metabox render callback.
+	 *
+	 * @param callable $callback The render callback to invoke.
+	 * @return string The captured output.
+	 */
+	private function capture_render( callable $callback ): string {
+		ob_start();
+		$callback( $this->post );
+		return (string) ob_get_clean();
+	}
```

PHPMD (2.15.0)

[error] The method capture_render is not named in camelCase. (undefined)

(CamelCaseMethodName)